pkg/awsManager/aws_manager_test.go

```diff
@@ -35,6 +36,10 @@ func setupDefaultMocks(t *testing.T) *mockSuite {
 	return mocks
 }
 
+func init() {
```

Comment:
These seem to have been broken for a very long time - should aim at adding the unit test to CI pipeline